app/controllers/api_controller/tags.rb

```diff
@@ -7,7 +7,8 @@ module Tags
     #
 
     def tags_query_resource(object)
-      object ? object.tags.where(Tag.arel_table[:name].matches "#{TAG_NAMESPACE}%") : {}
+      # object ? object.tags.where(Tag.arel_table[:name].matches "#{TAG_NAMESPACE}%") : {}
```

Comment:
Cannot use metaprogamming for Chargeback tags because you search by TAG_NAMESPACE (/managed)

But tag names are: 

```chargeback_rate/assigned_to/:object/id/:id```